CONTRIBUTING.md

```diff
@@ -67,6 +67,16 @@ For development you are going to want to checkout out the source. To get it, clo
     cd linguist/
     script/bootstrap
 
+To run Linguist from the cloned repository directory, you will need to generate the code samples first:
+
+    bundle exec rake samples
+
+If you are not running the tests below, re-run this command every time you modify any of the [samples][samples].
```

Comment:
Since `re-run` implies the user has already run the command, and the sentence makes the assumption the user modified a sample file, as opposed to a process or what-have-you.